webapp/pom.xml

```diff
@@ -50,6 +41,11 @@
 			<artifactId>ibis-adapterframework-akamai</artifactId>
 			<scope>runtime</scope>
 		</dependency>
+		<dependency>
```

Comment:
Now the profile `aws` is removed but the module `aws` is not part of the default modules list, if I see it correctly?

This module should be added to the default modules list or it will not be built!